tests/test_transforms/test_decomposition/test_change_points_trend_transform.py

```diff
@@ -5,6 +5,7 @@
 from sklearn.linear_model import LinearRegression
 
 from etna.datasets import TSDataset
+from etna.transforms.decomposition.base_change_points import RupturesChangePointsModel
```

Comment:
There should be a separate file with tests for BaseChangePointsModelAdapter and for RupturesChangePointsModel